services/user_api/queries.js

```diff
@@ -0,0 +1,82 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'admin',
+  host: 'localhost',
+  database: 'CS3219',
+  password: 'password',
+  port: 5432,
+})
+
+
+const getUsers = (request, response) => {
+    pool.query('SELECT * FROM users ORDER BY userId ASC', (error, results) => {
+      if (error) {
+        throw error
```

Comment:
Maybe do some simple error handling for all the APIs so that it doesn't crash
```suggestion
        response.status(500).json({ error: 'An internal server error occurred.' });
        return;
```

Reply:
Agreed. Will do so.